packages/ui/src/Common.ts

```diff
@@ -1878,16 +1878,28 @@ export interface PillProps {
   onClick: (enabled: boolean) => void;
 }
 
-export interface SegmentedControlProps {
+type BaseSegmentedControlProps = {
```

Comment:
This is a great idea. We probably could do this for split page as well since there are mobile props that don't apply to web

Reply:
Agreed! And definitely Field (returns the correct type in onChange) and a few others probably.